win/tty/wintty.c

```diff
@@ -77,6 +77,11 @@ extern short glyph2tile[];
 #define HUPSKIP_RESULT(RES) /*empty*/
 #endif /* ?HANGUP_HANDLING */
 
+#ifdef REALTIME_ON_BOTL
+#include <sys/select.h>
```

Comment:
should be guarded by #ifdef UNIX or so too

Reply:
ah right